packages/@glimmer/runtime/lib/environment.ts

```diff
@@ -287,15 +285,6 @@ export class EnvironmentImpl<Extra> implements Environment<Extra> {
     }
   }
 
-  iterableFor(ref: VersionedPathReference, inputKey: unknown): OpaqueIterable {
```

Comment:
Just wanted to note in the description that this is a breaking change as well (but in practice no consumers use it, I checked both Ember and Glimmer.js)